permissions/component-groovy-cps.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: "groovy-cps"
+github: "jenkinsci/workflow-cps-plugin"
+paths:
+  - "com/cloudbees/groovy-cps"
```

Comment:
TBD: See https://github.com/jenkinsci/workflow-cps-plugin/pull/612#discussion_r1010855691.

Reply:
IMO this is fine as is.